manifest.json

```diff
@@ -19,7 +18,8 @@
       "32": "images/logo/32.png",
       "48": "images/logo/48.png",
       "128": "images/logo/128.png"
-    }
+    },
+    "default_title": "DNSimple"
```

Comment:
What does this option do? It's not documented here: https://developer.chrome.com/apps/manifest

Reply:
Just a tooltip when you get over the button :)